packages/form_bloc/lib/src/blocs/field/field_bloc.dart

```diff
@@ -596,11 +673,34 @@ class MultiFieldBloc<ExtraData, TState extends MultiFieldBlocState<ExtraData>>
 
   Iterable<FieldBloc> get flatFieldBlocs => state.flatFieldBlocs;
 
-  /// See [FieldBloc.validate]
+  // ===========================================================================
+  // UTILITY
+  // ===========================================================================
+
+  /// {@macro form_bloc.isValuesChanged}
+  /// See [FormBloc.isValuesChanged]
+  bool get isValuesChanged => FormBlocUtils.isValuesChanged(flatFieldBlocs);
+
+  /// {@macro form_bloc.hasInitialValues}
+  /// See [FormBloc.hasInitialValues]
+  bool get hasInitialValues => FormBlocUtils.hasInitialValues(flatFieldBlocs);
+
+  /// {@macro form_bloc.hasUpdatedValues}
+  /// See [FormBloc.hasUpdatedValues]
+  bool get hasUpdatedValues => FormBlocUtils.hasUpdatedValues(flatFieldBlocs);
+
+  // ===========================================================================
+  // METHODS TO UPDATE STATE
+  // ===========================================================================
+
+  /// Validate the field.
+  /// If does not contain [fieldBlocs] returns `true` otherwise
+  /// if one of them returns `false` it fails
   @override
   Future<bool> validate() {
     if (state.flatFieldBlocs.isEmpty) return Future.value(true);
 
+    // TODO(Reviewer): Should we wait until all fieldBlocs have been validated?
```

Comment:
@WahdanZ  What do you think?

Reply:
Yes i agree with you we need to waill all of them.
but can you tell me about the other options what do you think about what pros and cons?

Reply:
Trying to implement scrolling to the first widget with an error it may result that you go ungluing not to the first widget but to a few below if the first ones have finished validating later.
For now, to create complexity, I will wait for the validation of all fields

(https://github.com/GiancarloCode/form_bloc/issues/254)